packages/palette/src/utils/useDidMount.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useState } from "react"
+
+export const useDidMount = (defaultMounted = false) => {
+  const [isMounted, toggleMounted] = useState(defaultMounted)
+
+  useEffect(() => {
+    toggleMounted(true)
+  }, [])
+
+  return isMounted
+}
```

Comment:
We'll want to add this hook to the docs and perhaps follow-up in Force as we have something similar there.

Reply:
👍 Will add a docs pass now